roles/kube-node-common/tasks/kubelet.yml

```diff
@@ -78,10 +78,20 @@
     state: present
     update_cache: yes
 
+- name: Download cri-tools
+  get_url:
+    url: https://github.com/kubernetes-sigs/cri-tools/releases/download/v{{ critools_version }}/{{ item }}-v{{ critools_version }}-linux-amd64.tar.gz
```

Comment:
Please, make it possible to overwrite the default download URL from a variable to allow the use of the role in environments where Internet access is controlled.

Something like a default value in `vars/main.yml` and an input variable in `defaults/main.yml` should do it.

Reply:
Good idea!

Reply:
I did it directly in the `default/main.yml` with a variable refactor (using maps variable instead of plane variables + lookup function).